lib/core/text/abstract_text.nit

```diff
@@ -602,7 +605,13 @@ abstract class Text
 	#     assert "{0.code_point}{1.code_point}{8.code_point}{31.code_point}{32.code_point}".escape_to_c == "\\000\\001\\010\\037 "
 	#
 	# The exceptions are the common `\t` and `\n`.
-	fun escape_to_c: String
+	fun escape_to_c: String do return escape_to_c_text_impl
+
+	# The original definition of `escape_to_c`
+	#
+	# This method is separated from `escape_to_c` mainly for testing purposes.
+	# Normally, you should not call this method explicitly nor redefine it.
+	protected fun escape_to_c_text_impl: String
```

Comment:
An alternative way might be to split `flat` in 2 modules: one for basic features and another for optimizations. In that case, the test must be a `test.sh` test in order to bypass nitunit’s imports.